worker/restore_map.go

```diff
@@ -758,6 +774,7 @@ func RunMapper(req *pb.RestoreRequest, mapDir string) (*mapResult, error) {
 				maxBannedNs = x.Max(maxBannedNs, ns)
 			}
 		}
+		glog.Infof("[MAP] Reading manifest %d done! dropAll: %v\n", manifest.BackupNum, dropAll)
```

Comment:
Wouldn't there be too many logs? Should be okay though.

Reply:
Yeah, should be okay